### ISSUE TITLE
Button icons are now shown with base href in html

### ISSUE THREAD

**Issue by Aleksandr-ru** [Alex-D/Trumbowyg#256]
Hello :)
If there is a ````<base href="...">```` element in HTML document, pointing with href to somewhere else than current url, there are no icons visible on buttons of the editor.

This is caused by ````<svg><use xlink:href="...">```` in buildBtn function.

My way to fix it is to add ````window.location```` to the href, for example add it to line 675:
`````
html: '<svg><use xlink:href="' + window.location + '#' + prefix + (btn.ico || btnName).replace(/([A-Z]+)/g, '-$1').toLowerCase() + '"/></svg>',
````
And line 738:
````
html: '<svg><use xlink:href="' + window.location + '#' + prefix + (btn.ico || btnName).replace(/([A-Z]+)/g, '-$1').toLowerCase() + '"/></svg>' + (btn.text || btn.title || t.lang[btnName] || btnName),
````
Or maybe ````window.location.href.split('#').shift()```` as a 100% solution
Thanks)

### Informations

**Browser + version**: Firefox 45 or any
**OS**: Windows 7 or any
**Resolution**: 1680x1050 or any


### How to reproduce the bug?
Make a page in a different path than Trumbowyg
Add Trumbowyg
Add  ````<base href="...">```` pointing to somewhere different than the page
See that there are no icons

**Comment by RespectTheCoder** [comment]
Can confirm that this solution works.

**Comment by SwithFr** [comment]
I can confirm too ;)

**Comment by Miztah87** [comment]
Thanks @Aleksandr-ru  , great solution.
 Works like a charm!

**Comment by lucidlemon** [comment]
uhm could we get this integrated by default? makes it a bit hard to work with browserify right now, as I don't want to manipulate files within the node_modules folder.

**Comment by Alex-D** [comment]
I need to update the doc for the v2. In fact, you can use the `svgPath` option, but that does not fix this bug. If you apply the OP bugfix you break the IE/Edge support.

**Comment by Alex-D** [comment]
Pretty patch: https://gist.github.com/leonderijke/c5cf7c5b2e424c0061d2

**Comment by Alex-D** [comment]
I've just work on a patch on the `develop` branch. If someone want to test, he is welcome :)

**Comment by Alex-D** [comment]
Fix'd in v2.1.0. See https://alex-d.github.io/Trumbowyg/documentation.html#svg-icons if you need it
